docs/api/web-contents.md

```diff
@@ -403,6 +403,12 @@ a meta tag:
 <meta name='theme-color' content='#ff0000'>
 ```
 
+Returns:
+
+* `event` Event
+* `color` string or `null`
+  * Theme color in format '#rrggbb'. `null` when theme color is not set.
```

Comment:
So we don't confuse the docs parser this needs to either be a new paragraph or on the same line as `color`

E.g.

```markdown
* `color` (String | null) - Theme color is in the format '#rrggbb'. It is `null` when no theme color is set
```